src/__a11y__/run-a11y-tests.ts

```diff
@@ -28,6 +28,7 @@ export default function runA11yTests(theme: Theme, mode: Mode, skip: string[] =
       const testFunction =
         [
           ...skip,
+          'theming/tokens',
```

Comment:
doesn't this regress our test coverage here? https://github.com/cloudscape-design/components/blob/7433543fcd2796d6398b67867d25b637c1a2b6de/src/__a11y__/a11y-dark.test.ts

Reply:
No. The page was artificially constructed to inspect theming application [example](https://d21d5uik3ws71m.cloudfront.net/components/ec6e5132b805605e94846be0200ea3053bec9365/index.html#/light/theming/tokens), and testing for accessibility would be counterproductive. The page was already excluded from VR a11y tests. It used to pass for classic as theming was applied by default (making all tiles green) and we now need to exclude it because I changed the behavior to only activate theming upon a click.